weblate/templates/keyboard-shortcuts.html

```diff
@@ -0,0 +1,166 @@
+{% load i18n %}
+
+<!-- Modal that shows available keyboard shortcuts  -->
+<div class="modal fade " id="shortcuts-modal" tabindex="-1" role="dialog" aria-labelledby="shortcutsModal">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{% trans " Close" %}">
+          <span aria-hidden="true">&times;
+        </button>
+        <h4 id="shortcutsModal" class="panel-title">{% trans "Available Shortcuts" %}</h4>
+      </div>
+      <div class="modal-body" id="shortcuts-table-container">
+        <table class="table table-striped">
+          <thead class="thead-dark">
+            <tr>
+              <th scope="col" id="t-head">{% trans "Shortcut" %}</th>
+              <th scope="col" id="t-head">{% trans "Action" %}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="shortcut">
+              <td><kbd>?</kbd></td>
+              <td>{% trans "Open available keyboard shortcuts"%}</td>
+            </tr>
+            <tr>
+              <td><kbd>Alt</kbd> + <kbd>Home</kbd></td>
+              <td>{% trans "Navigate to first translation in current search."%}</td>
+            </tr>
+            <tr>
+              <td><kbd>Alt</kbd> + <kbd>End</kbd></td>
+              <td>{% trans "Navigate to last translation in current search."%}</td>
+            </tr>
+            <tr>
+              <td>
+                <kbd>Alt</kbd> + <kbd>PageUp</kbd> {% trans "or" %} <br>
```

Comment:
We should provide context in these separators so that it is clear when translating:

```suggestion
                <kbd>Alt</kbd> + <kbd>PageUp</kbd> {% translate "or" context "between keyboard shortcuts" %} <br>
```